app/components/ui/notification-stream/component.js

```diff
@@ -42,7 +42,12 @@ export default Component.extend({
         const self = this;
 
         const event = self.get('selectedEvent');
-        self.get('store').findRecord('job', event.json.data.resource.jobs[0]).then(job => {
+        if (event.json.data.resource.jobs != null) {
```

Comment:
```suggestion
        if (event.json.data.resource.jobs && event.json.data.resource.jobs.length) {
```

This would throw an error for an empty array, as `[]` is not equal to `null`.
May be better to check for a truthy length instead, as that would guarantee that there is indeed at least one item in the list